client/me/developer/main.jsx

```diff
@@ -128,35 +109,57 @@ class Developer extends Component {
 
 	render() {
 		return (
-			<Main className="developer" wideLayout>
-				<PageViewTracker path="/me/developer" title="Me > Developer" />
-				<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
-				<NavigationHeader
-					navigationItems={ [] }
-					title={ this.props.translate( 'Developer Features' ) }
-					subtitle={ this.props.translate(
-						'Take WordPress.com further with early access to new developer features.'
+			<>
+				<Main className="developer" wideLayout>
+					<PageViewTracker path="/me/developer" title="Me > Developer" />
+					<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
+					<NavigationHeader
+						navigationItems={ [] }
+						title={ this.props.translate( 'Developer Features' ) }
+						subtitle={ this.props.translate(
+							'Take WordPress.com further with early access to new developer features.'
+						) }
+						className="developer__header"
+					/>
+
+					<form onChange={ this.props.submitForm }>
+						<FormFieldset
+							className={ classnames( 'developer__is_dev_account-fieldset', {
+								'is-loading': this.props.isFetchingUserSettings,
+							} ) }
+						>
+							<ToggleControl
+								disabled={ this.props.isFetchingUserSettings || this.props.isUpdatingUserSettings }
+								checked={ this.props.getSetting( 'is_dev_account' ) }
+								onChange={ this.handleToggleIsDevAccount }
+								label={ getIAmDeveloperCopy( this.props.translate ) }
+							/>
+						</FormFieldset>
+					</form>
+
+					<DeveloperFeatures />
+				</Main>
+
+				{ this.state.isVisibleDevSurveyNotice &&
+					ReactDOM.createPortal(
+						<DevSurveyNotice
```

Comment:
Could we avoid using acronyms please 🙂?
```suggestion
						<DeveloperSurveyNotice
```